modules/ROOT/nav.adoc

```diff
@@ -386,5 +386,10 @@
 **** xref:6.0-release-notes-core-changes.adoc[Core changes]
 **** xref:6.0-release-notes-premium-changes.adoc[Premium changes]
 **** xref:6.0-release-notes-known-issues.adoc[Known issues]
+*** TinyMCE 6.0.2
+**** xref:6.0.2-release-notes.adoc#overview[Overview]
+**** xref:6.0.2-release-notes.adoc#general-bug-fixes[Core changes]
+**** xref:6.0.2-release-notes.adoc#accompanying-premium-plugin-changes[Premium changes]
+**** xref:6.0.2-release-notes.adoc#known-issues[Known issues]
```

Comment:
These are in the wrong place based on 5.x, as we list the versions in newest -> oldest. As such these should be above 6.0 instead of below.

- [ ] Fix nav

Reply:
Should be fixed now. Thanks